lib/flappi/config.rb

```diff
@@ -3,17 +3,13 @@
 module Flappi
   class Config
 
-    attr_reader :definition_paths
+    attr_accessor :definition_paths
     attr_accessor :version_plan
     attr_accessor :doc_target_path
     attr_accessor :doc_target_paths
     attr_accessor :logger_level
     attr_reader :logger_target
 
-    def definition_paths=(v)
```

Comment:
This was a convenience method to take either a single path or multiple paths. Why delete it?

Reply:
So `definition_paths` is now a hash of version => path_or_array_of_paths ?